rancher/rancher-ui/applications/stacks/adding-balancers/index.md

```diff
@@ -10,20 +10,99 @@ After adding multiple services to your stack, you might have decided that you wa
+In our example from the [adding services section]({{site.baseurl}}/rancher/rancher-ui/applications/stacks/adding-services/), we only have 2 hosts in our environment, so therefore we can only create a maximum of 2 load balancers.
 
-We'll define the listening ports on the load balancer. These ports will be accessed publicly through through. You can define the protocol for each port. Currently, the load balancers have only one algorithm that is used by HAProxy, which is round robin. HAProxy is the software that is installed on our load balancers. 
+We'll define the listening ports on the load balancer. The source port is the port that the load balancer is listening on the host. You can decide if this source port will be accessible publicly (i.e. accessible outside of the host) or internally on the host. You can also choose the target port of the load balancer. The target port is the private port that traffic will be directed to the services. If no target port is defined, the source port will also be used as the target port. You can select the protocol for each listening port. 
 
-In our example, we'll use `8090` with the `http` protocol. 
+Let's use a source port of `8090`, target port of `80`, select `Public` access and select the `http` protocol. Next, let's add our target. We will select our wordpress service. Click on **Create**. 
 
-Next, we will select our target and port, which is our wordpress service. The available target(s) in the drop down list are any services within the stack. The target port is the private port that targets will use to communicate with the hosts. 
+Just like with services, the load balancer is not started until the user starts the service. You can individually start the load balancer by clicking **Start** or selecting **Start** in the dropdown menu. 
 
-In our example, we'll select our wordpress service and use port `80`.
+Now, let's see the load balancer in action. Click on a container name inside the load balancer. This will bring you to a detailed container page of the load balancer. Copy the IP address of the host, paste the IP into the web browser of your choice and add `:8090`. The Wordpress application will come up in the browser. 
 
+It's that easy to set up a load balancer with Rancher! 
+
+## Load Balancer Routing 
+
+Our load balancer has HAProxy software installed on the load balancer agent containers. The load balancer uses a round robin algorithm from HAProxy to select the target services. 
+
+### Internal Load Balancers
+
+With Rancher, you have the ability to create an internal load balancer by setting the listening port(s) to `Internal`. When the listening port is set to internal, then the source port of the load balancer will not be exposed on the host. Therefore, for any internal load balancer on a host, only services on the same host will be able to access this load balancer. Any services on a different host will **not** be able to access the load balancer.
```

Comment:
"will not be exposed" should be changed to "will not be published". "Expose" docker parameter means that the service exposes this particular port, w/o actually publishing it to the host.